autodoc/admin-api/data/admin-api.lua

```diff
@@ -478,6 +478,72 @@ return {
           ]],
         },
       },
+      ["/timers"] = {
+        GET = {
+          title = [[Retrieve runtime debugging info of Kong's timers]],
+          endpoint = [[<div class="endpoint post">/timers</div>]],
+          description = [[
+            Retrieve runtime stats data from [lua-resty-timer-ng](https://github.com/Kong/lua-resty-timer-ng).
+          ]],
+          response = [[
+            ```
+            HTTP 200 OK
+            ```
+
+            ```json
+            {
+                "flamegraph": {
+                    ...
+                },
+                "sys": {
+                    "running": 0,
+                    "runs": 7,
+                    "pending": 0,
+                    "waiting": 7,
+                    "total": 7
+                },
+                "timers": {
+                    "...": {
+                        "name": "...",
+                        "meta": {
+                            ...
+                        },
+                        "stats": {
+                            "finish": 2,
+                            "runs": 2,
+                            "elapsed_time": {
+                                "min": 0,
+                                "max": 0,
+                                "avg": 0,
+                                "variance": 0
+                            },
+                            "last_err_msg": ""
+                        }
+                    }
+                }
+            }
+            ```
+
+            * `flamegraph`: String-encoded timer-related flamegraph data.
+              You can use [brendangregg/FlameGraph](https://github.com/brendangregg/FlameGraph) to generate flamegraph svgs.
+            * `sys`: List the number of different type of timers.
+              * running: number of running timers.
```

Comment:
```suggestion
              * `running`: number of running timers.
```

Same for the next few lines.

Reply:
Thanks! done!